upgrade/run_upgrade_test.sh

```diff
@@ -47,15 +47,19 @@ function download_untar_istio_release() {
   local dir=${3:-.}
   # Download artifacts
   LINUX_DIST_URL="${url_path}/istio-${tag}-linux.tar.gz"
+  echo "${LINUX_DIST_URL}"
```

Comment:
Strictly speaking this is not needed as the script run with `set -x`

Reply:
yes, thanks pointing it out.